Controllers/TeamMembersController.cs

```diff
@@ -39,9 +48,19 @@ public async Task<IActionResult> Authenticate([FromBody] TeamMembers teammemberO
 
             var teammember = await _context.TeamMembers.FirstOrDefaultAsync(x => x.username == teammemberObj.username); // && x.Password == teammemberObj.Password);
 
+            var teammember_email = await _context.TeamMembers.FirstOrDefaultAsync(x => x.email == teammemberObj.username);
+
+
             if (teammember == null)
             {
-                return NotFound(new { Message = "User Not Found" });
+                if (teammember_email == null)
```

Comment:
Why is this check needed? Just curious

Reply:
Allows users to login with email or username